components/capi/src/fixed_decimal.rs

```diff
@@ -0,0 +1,40 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use fixed_decimal::FixedDecimal;
+
+/// Opaque type for use behind a pointer, is [`FixedDecimal`]
+///
+/// Can be obtained via [`icu4x_fixed_decimal_create()`] and destroyed via [`icu4x_fixed_decimal_destroy()`]
+pub type ICU4XFixedDecimal = FixedDecimal;
+
+#[no_mangle]
+/// FFI version of [`FixedDecimal`]'s constructors. This constructs a [`FixedDecimal`] of the provided
+/// `magnitude`.
+//
+// We can add additional constructors from strings, floats, etc as the need arises
+pub extern "C" fn icu4x_fixed_decimal_create(magnitude: i64) -> *mut ICU4XFixedDecimal {
+    let fd = FixedDecimal::from(magnitude);
+    Box::into_raw(Box::new(fd))
+}
+
+#[no_mangle]
+/// FFI version of [`FixedDecimal::multiply_pow10()`]. See its docs for more details.ICU4XFixedDecimal
+///
+/// Returns `true` if the multiplication was successful.
+pub extern "C" fn icu4x_fixed_decimal_multiply_pow10(
```

Comment:
Suggestion: Add `negate()` here as well, since it is the only way to get -0. Consider optionally adding `.signum()` as well. We should strive to have full API coverage.

Reply:
I feel like methods _reading_ from FixedDecimal aren't really that necessary in our public API for now.

My attitude here is more go go gradually and add stuff as necessary. But I added `negate()`